<!DOCTYPE html>
<html lang="en">

<head>
<script type="text/javascript" src="layout/head.js"></script>
</head>

<body>
<!-- Google Analytics -->
<!-- Global site tag (gtag.js) - Google Analytics -->
<script async src="https://www.googletagmanager.com/gtag/js?id=UA-141650531-1"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'UA-141650531-1');
</script>
<!-- /Google Analytics -->

<!--MoraLab banner-----------------------------------------------------------------------------------------> 
    <script type="text/javascript" src="layout/MoraLab_banner.js"></script>
<!--MoraLab banner-----------------------------------------------------------------------------------------> 

<!--Menu section------------------------------------------------------------------------------------------------------>
<script type="text/javascript" src="layout/menu.js"></script>
<!--Menu section------------------------------------------------------------------------------------------------------>

<!--Cards section------------------------------------------------------------------------------------------------------>
<section class="features3 cid-rsvlm0pUAH" id="features3-k">
    <div class="container">
        <div class="media-container-row">
            <div class="card p-3 col-12 col-md-6">
                <div class="card-wrapper">
                    <div class="card-img">
                        <img src="assets/images/gmu1.gif" alt="GMU" title="">
                    </div>
                    <div class="card-box">
                        <H3 class="card-title mbr-fonts-style display-7">Chromatin interaction hubs and their dynamics in human cells</H3>
                        <p class="mbr-text mbr-fonts-style display-7">
                        <br>Gene regulatory processes are some of the core processes that give each cell their identity. They are not only dependent on the genome sequence, epigenetic marks, and transcription factor action, but also on the three-dimensional structure of chromatin, which has been largely explored during the last decade thanks to chromosome conformation capture and other associated methodologies. Multiple chromatin interaction maps have been drafted and proven useful to explain gene activation and repression, and to find out the genes affected by the SNPs in regulatory elements. Interestingly, we have recently started to unravel even more complex chromatin interaction networks of promoters, enhancers, lncRNAs, nascent RNAs, phase-separated nuclear bodies, RNA polymerases, and other macromolecules, which has opened the possibility of understanding phenomena such as gene co-expression or the effects of genetic perturbations (either natural or engineered mutations). I am specially interested in chromatin hubs, densely connected subnetworks of the chromatin interaction network, which can explain genetic disease by linking SNPs to multiple genomic and epigenomic elements, explain DNA–DNA, protein–DNA, and lncRNA–DNA interaction specificity, or explain redundant mechanisms in terms of 3D co-localization. Chromatin hub disruption has been associated with multiple diseases, therefore my main goal is to identify chromatin hubs (from both HiC data and from computational prediction) and, particularly, to study the dynamics of healthy chromatin hubs, the dynamics of diseased chromatin hubs, and the dynamics and mechanisms of hub disruption.
                        <br>
                        <br>Among other studies: (i) We have written two important reviews in this area, the first one reviewing the biological and bioinformatic basis of chromatin interactions [1], and the second one doing the same for chromatin hubs [2]. (ii) We have also built a database and web platform called GREG that integrates multiple genomic and epigenomic data, using a graph database technology [3]. (iii) With GREG, we were able to explore the proteins and DNA elements interacting with COPD's SNPs and discover that such elements are functionally enriched on genes responsive to smoking and other functions related to the pathogenesis of the disease [3]. (iv) We have recently built a chromatin interaction analysis platform that combines all of the most popular software for chromatin interaction analysis (which we previously reviewed) in order to streamline such analyses [4]. (v) We are using such platform for an in-depth study of vascular endothelial cells [4]. Our current plans include: (v) Collaborative projects involving chromatin dynamics in endothelial cells and angiogenesis, as well as (vi) data collection and labeling for identified chromatin hubs from different technologies.
                        <br>
                        </p>
                        <h4 class="card-title mbr-fonts-style display-7">References:</h4>
                        <p class="mbr-text mbr-fonts-style display-7">
                        [1] MORA A, SANDVE G, GABRIELSEN O, and ESKELAND R (2015), In the loop: promoter–enhancer interactions and bioinformatics, Briefings in Bioinformatics, 17;6;980 <a href="https://doi.org/10.1093/bib/bbv097">[Website]</a><br>
                        [2] MORA A, HUANG X, JAUHARI S, JIANG Q, and LI X (2022), Chromatin Hubs: A biological and computational outlook, Computational and Structural Biotechnology Journal, 20;3796 <a href="https://doi.org/10.1016/j.csbj.2022.07.002">[Website]</a><br>
                        [3] MEI S, HUANG X, XIE C, and MORA A (2020), GREG—studying transcriptional regulation using integrative graph databases, Database, 2020, baz162 <a href="https://doi.org/10.1093/database/baz162">[Website]</a><br>
                        [4] Unpublished<br>
                        </p>
                    </div>
                </div>
            </div>
        </div>
    </div>
</section>
<!--Cards section------------------------------------------------------------------------------------------------------>

<!--Footer section------------------------------------------------------------------------------------------------------>
<script type="text/javascript" src="layout/footer.js"></script>
<!--Footer section------------------------------------------------------------------------------------------------------>
  
</body>
</html>
